### PR TITLE
Remove SECourses videos

### PR DESCRIPTION
**Pull request by hablaba**
In this change I remove videos posted by SECourses FurkanGozukara. The reason is that Furkan does not operate in the spirit of the open source community. I've frequently seen him both here and in sd-scripts repo spamming with posts to his videos as well as begging people to give their full commands for training to him. Recently he has been also spamming Reddit with posts of his trainings and claims that all of his information is "free" on Medium and YouTube but the reality is he gates the important parameters (learning rate, batch size, etc) behind his paid patreon.

I think it is misleading to link to his videos in this repo Readme given how he leeches from the community for settings while then turning around and paywalling his learnings. I have many other videos I can suggest, for example from AItrepeneur or others. What is more typical is a video should go into all details of the training, including relevant settings, while potentially paywalling things that just save time. For example AItrepeneur talks in depth about learning rate, dim, etc. and even shows examples, and the only thing he actually gates behind patreon are time saving scripts or reg images. 

The truth is many in the community don't even think the trainings by SECourses are even very good. So referencing him as an expert artificially props him up such that new people may pay him for advice. I think we should instead promote free and open source content where possible where people not only take and learn from the community but also contribute back without paywalling the important details.

Happy to link to the various GitHub issues and Reddit that describe what I am referencing and also I can provide some reference videos from other more open YouTubers, etc.

### CODE REVIEW

**Comment by FurkanGozukara** [comment]
@hablaba watch first those videos. they have shown all settings. so this PR must be discarded.  And stop promoting AItrepeneur . his paywalls are many times bigger than me. he got 3k paid members on Patreon

**Comment by hablaba** [comment]
Your newest videos and posts on reddit hide the important training parameters like learning rate. I realize these _older_ videos perhaps don't hide those values, but they essentially just act as promotion material for you and will cause someone to think "oh let me look at his latest videos since he's probably learned some things since then".

I only reference Aitrepeneur because he includes all values in his videos and his paywall is just for convenience scripts. You in contrast hide the learning rate in your latest posts and videos, along with other important settings. So you are essentially grifting off the community and the information other people share freely with you and then refusing to share your learnings back with the community.

**Comment by FurkanGozukara** [comment]
@hablaba you are talking without wathcing my videos

none of the videoes shared on readme has such paywall

those new parameters will be also shown in a big new video but it is not ready yet

to find them i did over 100 full fine tuning experimentation

**Comment by chavinlo** [comment]
I agree with OP. Furkan has been spamming his videos on TONS of repositories and sites (reddit, discords, etc) so much that its annoying.

**Comment by hablaba** [comment]
Once again, you misunderstand what I am saying @FurkanGozukara . My claim is not solely about these particular videos but about your behavior in general. I will post some links:

On reddit you post this: [Comparison Between SDXL Full DreamBooth Training (includes Text Encoder) vs LoRA Training vs LoRA Extraction - Full workflow and details in the comment](https://www.reddit.com/r/StableDiffusion/comments/18nxwt3/comparison_between_sdxl_full_dreambooth_training/) which contains 
[the comment](https://www.reddit.com/r/StableDiffusion/comments/18nxwt3/comment/kednkxg/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button).

In the post title you claim "full workflow". In the comment you say "I strongly suggest to read it on Medium. Open article - no paywall or anything." But here is the thing, there is a paywall: your patreon.

In searching the [medium article](https://medium.com/@furkangozukara/comparison-between-sdxl-full-dreambooth-training-vs-lora-training-vs-lora-extraction-44ada854f1b9), you don't post relevant configuration of your LoRA or Dreambooth training:

* Learning rate
* Batch size
* Optimizer
* LR scheduler
* BF16 / FP16
* Gradient checkpointing or not 
* Gradient accumulation
* Min SNR Gamma

Basically every relevant piece of config you leave out. So two claims are invalid: that this is the full workflow, and that there is no paywall. To get the config with these basic training settings that you beg everyone for all over GitHub Issues one must pay for your patreon. Or you also offer "private consulting". 

Here are some other posts where you also don't post your settings [[1]](https://www.reddit.com/r/StableDiffusion/comments/18uh0nn/dreambooth_sdxl_10_of_myself_vs_dalle_3_vs_sdxl/), [[2]](https://www.reddit.com/r/StableDiffusion/comments/18vb3fx/le_tour_jurassic_park_with_stable_diffusion_xl/), [[3]](https://www.reddit.com/r/StableDiffusion/comments/18segjh/generate_photos_of_yourself_in_different_cities/), [[4]](https://www.reddit.com/r/DreamBooth/comments/18myal5/trained_a_new_stable_diffusion_xl_sdxl_base_10/). I could post a lot more but... 

It's honestly a bit hard to navigate your [Reddit history ](https://www.reddit.com/user/CeFurkan/submitted/) because you literally spam so many subreddits so frequently that it's a bit hard to get beyond 30 days without a lot of scrolling. Interestingly though, [here is another one ](https://www.reddit.com/r/StableDiffusion/comments/17qy4hi/stable_diffusion_xl_sdxl_full_dreambooth_model_vs/)that I found from 2 months ago which requires a significant amount of scrolling where you also didn't provide settings. Apparently mods removed this post, not sure why. Probably due to you grifting.

I do recall another past post where you claimed that TE training was broken in sd-scripts. I can't find it now cause I'd have to scroll through 8 billion spam posts on reddit, but I believe this was the relevant sd-scripts issue: https://github.com/kohya-ss/sd-scripts/issues/890

We can see @FurkanGozukara contributing nothing to the issue but just writing random nonsense while devs and actual issue reporters discuss what is going on. He claimed TE training was completely broken on reddit, but clearly per the issue it was specific to the block LR options.

Here is a really interesting comment that showcases what I am describing: https://github.com/kohya-ss/sd-scripts/issues/890#issuecomment-1780199660

> can you share your command please?

Here he asks for someone to share their command so that he can benefit, yet he is unwilling to provide the same benefit back to the community. Instead his goal is to spam the community as much as possible to drive people to his Patreon where he has gated some of this basic information from his "learnings". 

There are a ton of other examples of @FurkanGozukara trolling both this GitHub repo, sd-scripts, as well as many other issues basically begging people for configs and contributing nothing to the actual conversation at hand. Then when it comes his turn to share with the community some basic settings (learning rate, batch size, etc) he claims he is sharing "free" "full workflow" "no paywall" medium articles and posts where he doesn't actually share anything important. How can we possibly evaluate your comparison of LoRA training to full fine tune if you can't even post a learning rate for each or other basic settings?

**Comment by hablaba** [comment]
I could post a lot more, but going through issues takes quite a while. If I find them I will post other examples. We can see though in sd-scripts @FurkanGozukara posts this: https://github.com/kohya-ss/sd-scripts/pull/736

Where he claims 

> I think this is the most comprehensive ever tutorial made for Kohya

And receives 6 downvotes. 

I'd agree with @Sj-Si assessment:

> First time I've seen a PR that was literally a self advertisement. Does this not seem a bit pretentious? Not only that but it is the first line in the Tips section. Man, what a laugh.

This guy doesn't contribute in any way but posts self advertisements in these repos as a way to get clients and patreon subscribers. It would be a different thing if some random person was like "wow I found this video so helpful, let me add it to the readme". No, it is pure self advertisement and marketing.

**Comment by FurkanGozukara** [comment]
@hablaba  you are still 100% unrelated talking about the readme of bmaltais/kohya_ss repo

The videos on Readme has no paywall and i spent at least 1 week for each one

they have full chapters and 100% accurate subtitles. which content producer does the same show me

waste your time with more useful stuff to contribute community it will be helpful to more people

**Comment by hablaba** [comment]
You're still not listening here. I am advocating that this well known open source repo shouldn't act as marketing or advertisement for a known spammer and essentially a leech. You troll around GitHub issues begging for configs and then turn around and release a bunch of spammy content without the actual relevant training parameters (as linked in my previous comment). Your _old_ videos contain the parameters but as of the last 2-3 months you have been putting everything behind Patreon while spamming various platforms and offering "private consulting". I already provided many links and I'm sure I'll provide more as I find the GitHub issues. 

Without this open source repo you wouldn't be able to even train your models or loras. This repo is free and contributed by the community. Why don't you want to also share your parameters in an open and free way? Regardless of if you shared the parameters or not, your general spammy content on Reddit and GitHub MRs (posting what are essentially ads and marketing materials) and annoying comments in GitHub issues is a problem that many others find issue with as well.

**Comment by FurkanGozukara** [comment]
@hablaba you are the spammer here. I have PHD in Computer Engineering. This is my google scholar : https://scholar.google.com/citations?user=_2_KAUsAAAAJ&hl=en

I am doing this full time job right now so stop trying to teach me how to do it.

Coqui just shut down itself do you know why? Because of spammers like you

image

**Comment by chavinlo** [comment]
> I have PHD in Computer Engineering.

you act like a 10 year old

> I am doing this full time job right now so stop trying to teach me how to do it.

get a real job rather than begging for money on the internet

> If you don't want to see any paywall things then pay my bills and i will do that. Again I repeat on this repo readme there is no such paywall videos. That is also why you are a spammer.

how tf is giving feedback on the repository spamming?
spamming is what you've been doing in the past few months linking your (garbage) videos on almost every ai-related site, people I know already know you for spamming and solely for that.

> Coqui just shut down itself do you know why? Because of spammers like you
> Coqui was the very best open source TTS
> You have no idea about Open Source community. You are just polluting it right now

you are a complete retard who doesn't even knows what the fuck happened with coqui
the model was too expensive to run to even have a profit and nobody wanted to pay coqui's overpriced services and since it was oss people just self-hosted it on other clouds
https://news.ycombinator.com/item?id=38437743
it has absolutely nothing to do with spammers or whathever shit you are pulling out your ass
you are trying so hard to turn the tables by calling us "spammers" that its obvious you want to avoid getting called that

keep it up and I will personally send someone to terminate your patreon and youtube so you can spam as hard as you did back in november '22 when you were just starting

**Comment by hablaba** [comment]
Let's try and keep things civil here. To summarize my main argument: I don't think it's really appropriate for @FurkanGozukara to be self promoting via open source GitHub repos. There are many issues both in this repo, sd-scripts, and others where he posts videos to his YouTube. He is incredibly spammy on Reddit as well, where you can see he posts his videos on all the ai subreddits constantly. As of the last few months, he has pushed to not only spamming the subreddits but gating basic information behind his Patreon (learning rate, batch size, dim/alpha, pretty much all basic settings). He is also just generally annoying in many random issues posted here and in sd-scripts where he isn't actually contributing but instead is either begging people for their configs, posting links to his videos, or asking when things are going to be merged on issues that were closed. 

I recommend that his videos are removed. In the place of his videos I'm sure we can find other videos to post, there are many that describe the same setup. There are also some great articles I've read that get into the setup. I'd be happy to post some links. I don't think in general it makes sense to allow people to self promote in an open source repo like this where so many people go when they first want to get setup training SD. I think that's true in particular when their behavior is incredibly spammy across different platforms and they are disingenous in their claims on "full workflow" "no paywall" etc.

Ultimately it is up to @bmaltais to decide what to do here, I just wanted to at least bring the issue to his attention.

**Comment by bmaltais** [comment]
This is a difficult subject. On the one hand I totally understand the position regarding YouTube video that are used to monetize using private membership to obtain config files… even if the video content provide all the necessary config as part of the video.

i agree that the project should not be used to facilitate monetization as I am not monetizing my own project.

i will therefore revive all video from the project that are not my own. I am sure users can find the same video listed on the readme by searching on google or YouTube.

I will wait a few days to allow more discussion and see if further considerations should be included to confirm or infirm the removal of all YouTube links.

**Comment by FurkanGozukara** [comment]
@bmaltais the videos on the readme has no such paywall. everything and config is shared. you can watch and verify. by the way i never made any money from the project. currently the config that i haven't shared publicly yet is totally related to training hyper parameters which i spent money and time to find out. i did over 100 full dreambooth trainings.

i will also share them in a new big video but not ready yet. i am a big supporter of your project and suggesting everyone to use it.

**Comment by bmaltais** [comment]
I have updated the dev branch with the proposed readme that removed all tutorial video links. I am certain users will find what they need on YouTube or other platforms like Reddit or civitai. This should remove any perceived ambiguity about the tutorials that were shown on the repo. I trust this is a fair solution for everyone.

**Comment by FurkanGozukara** [comment]
> I have updated the dev branch with the proposed readme that removed all tutorial video links. I am certain users will find what they need on YouTube or other platforms like Reddit or civitai. This should remove any perceived ambiguity about the tutorials that were shown on the repo. I trust this is a fair solution for everyone.

sure it is ok. keep up amazing work.

**Comment by hablaba** [comment]
Thank you @bmaltais. I appreciate your consideration on this subject and I agree that seems like an appropriate action to take. Thank you for your work here.

**Comment by GrigoryEvko** [comment]
Oh, I just realized @bmaltais and this spamming prick are 2 different people. It was a bad idea to add a bunch of videos from this weirdo to such a popular repo in the first place. I'm glad that it's gone now.
